sdk/inc/azure/core/az_context.h

```diff
@@ -122,17 +119,18 @@ AZ_NODISCARD AZ_INLINE bool az_context_has_expired(az_context const* context, in
 }
 
 /**
- * @brief az_context_get_value walks up this az_context node's parents until it find a node whose
- * key matches the specified key and returns the corresponding value.
+ * @brief Walks up this #az_context node's parents until it find a node whose key matches the
+ * specified key and returns the corresponding value.
  *
- * @param[in] context The az_context node in the tree where checking starts.
- * @param[in] key A pointer to the key to be scanned for
- * @param[in] out_value A pointer to a 'void*' that will receive the key's associated value if the
- * key is found
- * @return  #AZ_OK if the key is found
+ * @param[in] context The #az_context node in the tree where checking starts.
+ * @param[in] key A pointer to the key to be scanned for.
+ * @param[out] out_value A pointer to a `void const*` that will receive the key's associated value
+ * if the key is found.
+ * @return  #AZ_OK if the key is found.
  *          #AZ_ERROR_ITEM_NOT_FOUND if no nodes are found with the specified key.
  */
-AZ_NODISCARD az_result az_context_get_value(az_context const* context, void* key, void** out_value);
+AZ_NODISCARD az_result
+az_context_get_value(az_context const* context, void const* key, void const** out_value);
```

Comment:
@RickWinter - please review this const-ness change with other `az_context` changes you might have, as part of your follow up.